src/slider/foundation.tsx

```diff
@@ -171,45 +324,53 @@ export const useSliderFoundation = (
   useEffect(() => {
     if (props.discrete !== undefined) {
       // @ts-ignore unsafe private variable access
-      foundation.isDiscrete_ = props.discrete;
+      foundation.isDiscrete = props.discrete;
     }
 
     if (props.discrete && foundation.getStep() === 0) {
       foundation.setStep(1);
     }
   }, [props.discrete, foundation]);
 
+  // minRange
+  useEffect(() => {
+    if (props.minRange !== undefined) {
+      foundation.setMinRange(Number(props.minRange));
+    }
+  }, [props.minRange, foundation]);
+
   // displayMarkers
   useEffect(() => {
     // @ts-ignore unsafe private variable access
-    const hasTrackMarker = foundation.hasTrackMarker_;
+    const hasTickMarks = foundation.hasTickMarks;
     if (
       props.displayMarkers !== undefined &&
-      props.displayMarkers !== hasTrackMarker
+      props.displayMarkers !== hasTickMarks
     ) {
       // @ts-ignore unsafe private variable access
-      foundation.hasTrackMarker_ = props.displayMarkers;
-      window.requestAnimationFrame(() => foundation.setupTrackMarker());
+      foundation.hasTickMarks = props.displayMarkers;
+      foundation.setHasTickMarks(props.displayMarkers);
     }
   }, [props.displayMarkers, foundation]);
 
   // bugfix
-  useEffect(() => {
-    // Fixes an issue where synthetic events were being
-    // accessed in the Foundation and causing an error
-    // @ts-ignore unsafe private access
-    const existinghandleDown_ = foundation.handleDown_.bind(foundation);
-
-    // @ts-ignore unsafe private access
-    foundation.handleDown_ = (evt: React.SyntheticEvent<any>) => {
-      evt.persist();
-      existinghandleDown_(evt);
-    };
-  }, [foundation]);
+  // useEffect(() => {
```

Comment:
Remove commented out code

Reply:
Decided to put it back, as it probably was fixing some bug that I am unaware of...